README.adoc

```diff
@@ -12,11 +12,11 @@
 :projectid: kubernetes-microprofile-config
 :page-layout: guide
 :page-duration: 15 minutes
-:page-releasedate: 2018-09-24
+:page-releasedate: 2018-10-12
 :page-description: Externalize configuration and use Kubernetes ConfigMaps and Secretes to configure your microservices.
-:page-tags: ['microservices', 'Kubernetes', 'Docker', 'containers', 'kubectl', 'Minikube', 'Microprofile', 'mpConfig', 'ConfigMaps', 'Secrets']
+:page-tags: ['microservices', 'Kubernetes', 'Docker', 'containers', 'kubectl', 'Minikube', 'MicroProfile', 'Config', 'ConfigMaps', 'Secrets']
 :page-permalink: /guides/{projectid}
-:page-related-guides: ['docker', 'istio', 'kubernetes']
+:page-related-guides: ['kubernetes', 'microprofile-config']
```

Comment:
Btw, I think the kube-intro guide projectid is "kubernetes-intro", you need this value for the related guide data.